### PR TITLE
Mobile fixes

### PR DESCRIPTION
**Pull request by TobiTenno**
### Warframe Hub Pull Request
---
---

**Summary (short):** Force 100% width on mobile widths

---
**Description:** Use a mediaquery for screen size to set column widths

---
**Fixes issue (include link):** #81 

---
**Mockups, screenshots, evidence:** 
image

---

(Check one)
- [x] Issue fix
- [ ] Suggestion fulfillment

### CODE REVIEW

**Comment by TobiTenno** [comment]
image

image

Could we re-packery on the last one on click?

File: assets/js/main.js [hunk 1]
```diff
@@ -221,6 +224,7 @@ function updateVoidTraderInventory() {
         const itemString = `<tr><td>${currentItem.item}</td>` +
                   `<td>${currentItem.ducats}</td><td>${currentItem.credits}</td></tr>`;
         $('#voidTraderInventoryContent').append(itemString);
+        $('a[href$="#voidTraderInventoryPanel"').click(() => pkry.packery());
```
Comment (Tithen-Firion) [inline assets/js/main.js]: Shouldn't that be just `updateGrid()`?

Comment (TobiTenno) [reply assets/js/main.js]: other sources said packery, but i'll try that instead.

Comment (Tithen-Firion) [reply assets/js/main.js]: No, I mean just `updateGrid()`. That's function exposed from Packery closure.

File: assets/js/main.js [hunk 2]
```diff
@@ -1092,6 +1087,12 @@ function update() {
   setTimeout(update, 30000);
 }
 
+let resizeId;
+$('.resize-trigger').click(() => {
```
Comment (Tithen-Firion) [inline assets/js/main.js]: Binds to 0 elements, `.resize-trigger` doesn't exist yet. [This](https://stackoverflow.com/a/33092954/2428152) might help with timeout.

Comment (TobiTenno) [reply assets/js/main.js]: _fek_ i just need to add it on ready instead of immediate.

Comment (TobiTenno) [reply assets/js/main.js]: yeah, that should help as well

**Comment by TobiTenno** [comment]
@EricSihaoLin thoughts?

**Comment by EricSihaoLin** [comment]
@TobiTenno Not sure if you want to fix this too:
image

**Review by EricSihaoLin — APPROVED** [review]

**Comment by TobiTenno** [comment]
that one's a bit weirder. is it doing that on your device? i only saw it get like that with crazy small widths

**Pull request merged**